scripts/mkcloud

```diff
@@ -332,8 +332,8 @@ function _lvcreate()
 
     # first: create on the PV device (spread IO)
     # fallback: create in VG (if PVs with different size exist)
-    lvcreate -n $lv_name -L ${lv_size}G $lv_vg $lv_pv || \
-        safely lvcreate -n $lv_name -L ${lv_size}G $lv_vg
+    lvcreate -n $lv_name --wipesignatures y --yes -L ${lv_size}G $lv_vg $lv_pv || \
+        safely lvcreate -n $lv_name --wipesignatures y --yes -L ${lv_size}G $lv_vg
```

Comment:
We have mkcloud1 with SLE-12 and mkcloud2 with SLE-11-SP4 so that we can make sure that mkcloud supports a wide range of host OSes.
can we maybe auto-detect the existence of the --yes param and only use it when available?